include/zephyr/dt-bindings/dma/rpi_pico_dma.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_RPI_PICO_DMA_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_RPI_PICO_DMA_H_
+
+/*
+ * Use lower 6-bit of inverted DREQ value for `slot` cell.
+ * Need to be able to work for memory-to-memory transfer
+ * with zero, which is the default value.
+ */
+#define RPI_PICO_DMA_SLOT_TO_DREQ(s) (~(s)&0x3F)
```

Comment:
Maybe some type casting/assertion before the value is inverted? This would give inconsistent results for various types

Reply:
DTS also use this file. Cannot casting here.